docs/lob_guide/src/asciidoc/_chapters/work_with_lob.adoc

```diff
@@ -0,0 +1,738 @@
+
+* If a column is declared with the `STORAGE 'External'` attribute, the `STRINGTOLOB` or `FILETOLOB` functions cannot be used to insert data into that column. 
++
+Similarly, if a column is declared without the `STORAGE 'External'` attribute, the `EXTERNALTOLOB` function cannot be used to insert/update data into that column.
++
+That is, once the storage for a LOB column is declared at CREATE time, the attribute is set and cannot be modified. 
+
+[#select column from a sql table containing lob columns]
+== Select Column from a SQL Table Containing LOB Columns
+
+Selecting a row from a table will give back the lob handle for the lob column. 
+
+NOTE: The entry in each lob column in the SQL table only contains the LOB handle.
+
+Once the LOB handle has been retrieved, it can be used to retrieve the actual lob data from HDFS.
+
+[#syntax]
+=== Syntax
+
+```
+SELECT lob_query-expr[, lob_query-expr] FROM table-name
+```
+
+```
+lob_query_expr is: 
+lobtostring(lob column)      |
+lob column
+```
+
+[#examples]
+=== Examples
+
+* This example selects c2 from table tlob1 and returns the lob handle of c2.
++
+```
+SELECT c2 FROM tlob1;
+C2
+----------------------------------------------------------------------------------------------------
+
+LOBH00000200010423909193650389683319694857010382259683718212310961182290216021"TRAFODION"."SEABASE" 
+
+--- 1 row(s) selected.
+```
+
+[#extract lob data from a sql table containing lob columns]
+== Extract LOB Data from a SQL Table Containing LOB Columns 
+
+Extract lob data can be done in two ways:
+
+* Extract lob data into a file for a given lob handle
++
+Extract from a LOB column straight into an on-platform linux or HDFS file in one shot. 
+
++
+Trafodion engine will take care of buffering the data and retrieve the lob data into the target file.
+
+* Extract lob data into a user specified buffer
++
+Extract from a LOB column into a target user buffer of a specified size. 
++
+Trafodion engine will retrieve exactly the amount of requested data. 
++
+The user/application will be responsible for redriving the extract until EOD. 
++
+The extracted buffer data can then be written to files or another location as the application chooses. 
+
+[#extract lob data into a file for a given lob handle]
+=== Extract Lob Data into a File for a Given Lob Handle 
+
+[#syntax]
+==== Syntax
+
+```
+EXTRACT LOBTOFILE (LOB 'lob handle as quoted string', 'filename URI format' [OPTION]) 
+```
+
+```
+OPTION is:                                         
+[, TRUNCATE]
+[, CREATE, TRUNCATE]
+[, APPEND]   
+[, CREATE, APPEND]
+```
+
+[#semantics]
+==== Semantics
+
+* `TRUNCATE`
++
+If the target file exists, Trafodion will truncate and write to it.
++
+If the target file does not exist, an error will be raised.
+
+* `CREATE, TRUNCATE`
++
+If the target file exists, Trafodion will truncate and write to it. 
++
+If the target file does not exist, create a table, then write to it. 

```

Comment:
Suggest "If the target file does not exist, Trafodion will create it and write to it." (I assume Trafodion is creating the file here and not creating a table somewhere.)

Reply:
Thanks Dave, I've corrected. :)